WordPress/Classes/ViewRelated/Post/PostListConflictResolver.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+
+struct PostListConflictResolver {
```

Comment:
I guess the conflict resolver itself doesn't need to know about Post List. What do you think?

We can give a view controller to it, but it doesn't necessarily need to be a `PostListViewController`. It could be any `UIViewController`.

Reply:
Agreed. I could see this being used on the Post editor, as well.